doc/source/science_guide/sg_dynamics.rst

```diff
@@ -261,40 +272,57 @@ This parameterization for the seabed stress is described in
 :cite:`Lemieux16`. It assumes that the largest keel draft varies linearly with the mean thickness in a grid cell (i.e. sea ice volume). The :math:`C_b` coefficients are expressed as
 
 .. math::
-   C_b= k_2 \max [0,(h_u - h_{cu})]  e^{-\alpha_b * (1 - a_u)} (\sqrt{u^2+v^2}+u_0)^{-1}, \\
+   C_b= k_2 \max [0,(h - h_{c})]  e^{-\alpha_b * (1 - a)} (\sqrt{u^2+v^2}+u_0)^{-1}, \\
    :label: Cb
 
 where :math:`k_2` determines the maximum seabed stress that can be sustained by the grounded parameterized ridge(s), :math:`u_0`
 is a small residual velocity and :math:`\alpha_b` is a parameter to ensure that the seabed stress quickly drops when
-the ice concentration is smaller than 1. In the code, :math:`k_2 \max [0,(h_u - h_{cu})]  e^{-\alpha_b * (1 - a_u)}` is defined as
-:math:`T_b`. The quantities :math:`h_u`, :math:`a_{u}` and :math:`h_{cu}` are calculated at
-the 'u' point based on local ice conditions (surrounding tracer points). They are respectively given by
+the ice concentration is smaller than 1. In the code, :math:`k_2 \max [0,(h - h_{c})]  e^{-\alpha_b * (1 - a)}` is defined as
+:math:`T_b`. 
+
+On the B grid, the quantities :math:`h`, :math:`a` and :math:`h_{c}` are calculated at
+the U point and are referred to as :math:`h_u`, :math:`a_{u}` and :math:`h_{cu}`. They are respectively given by
 
 .. math::
    h_u=\max[v_i(i,j),v_i(i+1,j),v_i(i,j+1),v_i(i+1,j+1)], \\
    :label: hu
 
 .. math::
-   a_u=\max[a_i(i,j),a_i(i+1,j),a_i(i,j+1),a_i(i+1,j+1)]. \\
+   a_u=\max[a_i(i,j),a_i(i+1,j),a_i(i,j+1),a_i(i+1,j+1)], \\
    :label: au
 
 .. math::
    h_{cu}=a_u h_{wu} / k_1, \\
    :label: hcu
 
-where the :math:`a_i` and :math:`v_i` are the total ice concentrations and ice volumes around the :math:`u` point :math:`i,j` and
+where the :math:`a_i` and :math:`v_i` are the total ice concentrations and ice volumes around the U point :math:`i,j` and
 :math:`k_1` is a parameter that defines the critical ice thickness :math:`h_{cu}` at which the parameterized
 ridge(s) reaches the seafloor for a water depth :math:`h_{wu}=\min[h_w(i,j),h_w(i+1,j),h_w(i,j+1),h_w(i+1,j+1)]`. Given the formulation of :math:`C_b` in equation :eq:`Cb`, the seabed stress components are non-zero only
 when :math:`h_u > h_{cu}`.
 
+As :math:`u` and :math:`v` are not collocated on the C grid, :math:`T_b` is calculated at E and N points. For example, at the E point, :math:`h_e`, :math:`a_{e}` and :math:`h_{ce}` are respectively
+
+.. math::
+   h_e=\max[v_i(i,j),v_i(i+1,j)], \\
+   :label: he
+
+.. math::
+   a_e=\max[a_i(i,j),a_i(i+1,j)], \\
+   :label: ae
+
+.. math::
+   h_{ce}=a_e h_{we} / k_1, \\
+   :label: hce
+
+where :math:`h_{we}=\min[h_w(i,j),h_w(i+1,j)]`. Similar calculations are done at the N points. 
```

Comment:
I'd like to review the seabed stress section in readthedocs, when available -- the notation is a bit difficult to follow here.

Reply:
I agree...I would like to work on it again.